requirements/requirements.txt

```diff
@@ -56,44 +57,45 @@ jinja2==2.10
 jsonpickle==0.9.6         # via asgi-amqp
 jsonschema==2.6.0
 kombu==4.2.1              # via asgi-amqp, celery
-lxml==4.2.3
+lxml==4.2.3               # via xmlsec
 markdown==2.6.11
-MarkupSafe==1.0           # via jinja2
+markupsafe==1.0           # via jinja2
 more-itertools==4.1.0     # via irc, jaraco.functools, jaraco.itertools
 msgpack-python==0.5.5     # via asgi-amqp
 netaddr==0.7.19           # via pyrad
 oauthlib==2.0.6           # via django-oauth-toolkit, requests-oauthlib, social-auth-core
 ordereddict==1.1
 pexpect==4.6.0
+pkgconfig==1.4.0          # via xmlsec
 psutil==5.4.3
 psycopg2==2.7.3.2
 ptyprocess==0.5.2         # via pexpect
-pyasn1-modules==0.2.1     # via service-identity
-pyasn1==0.4.2             # via pyasn1-modules, service-identity
+pyasn1-modules==0.2.1     # via python-ldap, service-identity
+pyasn1==0.4.2             # via pyasn1-modules, python-ldap, service-identity
 pycparser==2.18           # via cffi
 pygerduty==0.37.0
 pyjwt==1.6.0              # via social-auth-core, twilio
+pyopenssl==19.0.0         # via service-identity
 pyparsing==2.2.0
 pyrad==2.1                # via django-radius
-pysocks==1.6.8
+pysocks==1.6.8            # via twilio
 python-dateutil==2.7.2
 python-ldap==3.1.0        # via django-auth-ldap
-django-cors-headers==2.4.0
 python-logstash==0.4.6
 python-memcached==1.59
-python3-openid>=3.0.10      # via social-auth-core
 python-radius==1.0
+python3-openid==3.1.0     # via social-auth-core
 python3-saml==1.4.0
 pytz==2018.5              # via celery, django, irc, tempora, twilio
 pyyaml==3.12              # via djangorestframework-yaml
 redbaron==0.6.3
 requests-futures==0.9.7
 requests-oauthlib==0.8.0  # via social-auth-core
-requests[security]==2.15.1
+requests==2.15.1
```

Comment:
What's this change about?

Reply:
Unsure - this is the file that was generated after following the steps in the readme

Reply:
Let's change this back then.

Reply:
alright - I'll change this back manually. Should I do the same for `docutils`? the auto-generator took that out.

Reply:
Hrrrrm cc @chrismeyersfsu was it you who added the `docutils`?

Reply:
You can remove `docutils` we pre-install it in the Makefile so that `python-daemon` is happy.

Reply:
that makes sense - I'll go ahead and remove it, then.

Reply:
I don't think these changes have much to do with the library I'm adding for the credential plugin backend so I've moved all of this to it's own PR: https://github.com/ansible/awx/pull/3111.